core/templates/pages/donate-page/donate-page.module.ts

```diff
@@ -78,13 +86,8 @@ const downgradedModule = downgradeModule(bootstrapFn);
 
 declare var angular: ng.IAngularStatic;
 
-angular.module('oppia', [
-  'dndLists', 'headroom', 'infinite-scroll', 'ngAnimate',
-  'ngAudio', require('angular-cookies'), 'ngJoyRide', 'ngMaterial',
-  'ngSanitize', 'ngTouch', 'pascalprecht.translate',
-  'toastr', 'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate',
-  downgradedModule
-])
+angular.module('oppia').requires.push(downgradedModule);
+angular.module('oppia')
   // This directive is the downgraded version of the Angular component to
   // bootstrap the Angular 8.
   .directive(
```

Comment:
I'm not sure but can we remove this service-bootstrap from this module now @bansalnitish?

Reply:
Removed and checked it's working fine.

Reply:
@bansalnitish I think we can remove \<service-bootstrap\> too from [donate-page.mainpage.html](https://github.com/bansalnitish/oppia/blob/angularcomponent/core/templates/pages/donate-page/donate-page.mainpage.html)

Reply:
I just checked removing from the mainpage.html too. I'm getting an error `Error: Template parse errors:
Can't bind to 'ngif' since it isn't a known property of 'div'. ("`.

Reply:
Marking this as done, I've removed the service-bootstrap.